cargo-dist/src/main.rs

```diff
@@ -220,7 +221,25 @@ fn cmd_init(cli: &Cli, args: &InitArgs) -> Result<(), miette::Report> {
     do_init(&config, &args)
 }
 
-fn cmd_generate_ci(cli: &Cli, _args: &GenerateCiArgs) -> Result<(), miette::Report> {
+fn cmd_generate(cli: &Cli, args: &GenerateArgs) -> Result<(), miette::Report> {
+    let modes = if args.mode.is_empty() {
+        vec![GenerateMode::Ci]
+    } else {
+        args.mode.clone()
+    };
```

Comment:
I'd prefer that this be resolved deeper in the code. `generate`'s library impl should be reading the full config (probably just building a DistGraph) and determining which generate backends are in the "universe" (e.g. "they have ci=github, so I want to generate github ci", "they have installer=msi, so i want to generate wix files", ...).

If the input is an empty array, that's basically saying "no filters, generate everything". If the input isn't empty, that says "do the intersection of everything and this list".